nftables.go

```diff
@@ -91,17 +92,29 @@ func newInternal(family Family, table string, execer execer) (Interface, error)
 		return nil, fmt.Errorf("could not find nftables binary: %w", err)
 	}
 
-	cmd := exec.Command(nft.path, "--check", "add", "table", string(nft.family), nft.table)
-	_, err = nft.exec.Run(cmd)
+	cmd := exec.Command(nft.path, "--version")
```

Comment:
Nothing wrong here, but the comment for this function above doesn't make sense:
```
// for unit tests
```
??

(I don't know howto comment on lines not in the diff)

Reply:
> (I don't know howto comment on lines not in the diff)

You can't. You have to [randomly comment somewhere else and then apologize for it](https://github.com/kubernetes/kubernetes/pull/122724#discussion_r1465251869). :roll_eyes: